astronomer/airflow/version_check/templates/update-available.html

```diff
@@ -1,27 +1,43 @@
-{% if current_user.is_authenticated and cea_update_available %}
-  {% set level = "error" if cea_update_available.level == "critical" else "warning" %}
-  <div class="alert alert-{{ level }} ac-update-notice">
-    {% if "can_dismiss" | is_item_visible("UpdateAvailable") %}
-      <button data-href="{{ url_for('UpdateAvailable.dismiss', version=cea_update_available.version) }}" title="Ignore this update, and hide this message from all users">&times; Ignore this update</button>
+{% if current_user.is_authenticated %}
+  {% if cea_update_available %}
```

Comment:
```suggestion
{% if current_user.is_authenticated and cea_update_available %}
```

Let's undo this change, keep the nesting down a bit.

Reply:
I wanted to use the condition `current_user.is_authenticated` for both update notice and  EOL notice.